api/models.py

```diff
@@ -52,6 +52,7 @@ class UserPreferences(models.Model):
     show_beta_interface = models.BooleanField(default=True)
     created_date = models.DateTimeField(auto_now_add=True)
     modified_date = models.DateTimeField(auto_now=True)
+    airport_ui = models.NullBooleanField(default=None, null=True)
```

Comment:
the `NullBoolean` here is that if you've never set it (so it's _null_) then they haven't logged in since we made the change. It's a bit arcane, but it's one way to treat queries for "how many have logged in, and set it back to `airport_ui`?"